src/bot/mtproto-api/bot/scripts/sms-auth/base-mtproto.ts

```diff
@@ -45,7 +45,8 @@ export class BaseMtProto {
     const clientParams: TelegramClientParams = {
       connectionRetries: 5,
     };
-    const client = new TelegramClient(session, TELEGRAM_APP_ID, TELEGRAM_API_HASH, clientParams);
+
+    const client = new TelegramClient(session ?? "", TELEGRAM_APP_ID, TELEGRAM_API_HASH, clientParams);
```

Comment:
Empty string always seems wrong.

Reply:
It needs to be an empty string if no session string exists, the empty string here is assigned to within that package and we store the filled string but it's necessary to pass one.

Reply:
Empty string is truthy doesnt seem right to ever pass an empty string as a legitimate value

Reply:
```ts
import { TelegramClient } from "telegram";
```
We have no control over this mate, it's the pkg that's running the show in this regard